Revit_Engine/Query/ZJustifiedCurve.cs

```diff
@@ -0,0 +1,61 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Reflection.Attributes;
+using System.ComponentModel;
+using BH.oM.Geometry;
+using BH.oM.Physical.Elements;
+using BH.Engine.Physical;
+
+namespace BH.Engine.Adapters.Revit
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Returns the driving curve of the given IFramingElement adjusted based on the 'z Justification' parameter value pulled from Revit.")]
+        [Input("element", "The IFramingElement to query the in Revit defining location line of.")]
+        [Output("curve", "The in Revit geometry defining location line of the element.")]
+        public static ICurve ZJustifiedCurve(this IFramingElement element)
+        {
+            string zJustification = element.GetRevitParameterValue("z Justification").ToString();
+
+            zJustification = zJustification.ToLower();
```

Comment:
You need extra null checks here to make sure the code will not crash:
```suggestion
            if (element == null)
                return null;
                
            string zJustification = element.GetRevitParameterValue("z Justification")?.ToString();
            zJustification = zJustification?.ToLower();
```